opt_PXL.ino

```diff
@@ -283,13 +297,12 @@ void setupLEDs() {
 }
 
 void setupOLED(){
-  Serial.begin(9600);

```

Comment:
Was this removed purposefully?

Reply:
This was suggested by @per1234  to save some memory space and everything seems to work with the replaced code.

Reply:
I believe you're referring to my suggestion override of the Arduino core library's `serialEventRun()` at https://github.com/woodsy900/DIY_PIXELSTICK_Arduino_Nano/issues/1#issuecomment-429491784, which you implemented at line 92. So that's different than this removal of `Serial.begin(9600);` but if you're not using Serial then certainly it's not necessary.